### DISCUSSION TITLE
scaledobject.sh

### DISCUSSION THREAD

**Discussion by pavan-chakka** [kedacore/keda#3038] [Q&A / Need Help] [answered]
Hello,

After deploying azure function in AKS cluster with KEDA on it, it is creating scaledobject.sh. what is the command to see that scaledobject.sh file.

**Comment by tomkerkhove** [comment, marked as answer]
`kubectl get scaledobjects` should help you.

**Reply by pavan-chakka** [reply]
Thank you Tom, I tried that earlier, but that is just giving list of scaled objects in our namespace. Am looking for scaledobject.keda.sh script file that gets created automatically when we do a deployment using 'func kuberentes deploy'. Also, want to know the way to edit and redeploy using it. I tried with with below command, and I believe that is scaledobject.keda.sh file. But, can you please advise how to make change to that in case if I have to change default 'cooldownperiod' property from 300 to 180.

k describe scaledobject <scaledobjectname>

**Reply by tomkerkhove** [reply]
That's a question for https://github.com/Azure/azure-functions-core-tools.

It deploys everything for you, but if you want to have the YAML instead, you can add `--dry-run`.

**Reply by pavan-chakka** [reply]
ok Thanks.
